subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -142,34 +161,138 @@ struct CeLoginRc
         mComponent(0 == reasonParm ? 0 : componentParm), mReason(reasonParm)
     {}
 
-    operator uint16_t() const
+    inline operator uint16_t() const
     {
         // Format manually to allow for consistant behavior on Big vs Little
         // Endian systems
         return (uint8_t)mComponent << 8 | (uint8_t)mReason;
     }
 
+    inline bool isSuccess() const
+    {
+        return Success == (*this);
+    }
+
     uint8_t mComponent;
     uint8_t mReason;
 };
 
-CeLoginRc getServiceAuthorityV1(
+struct AcfUserFields
+{
+    AcfUserFields() { clear(); }
+
+    void clear()
+    {
+        memset(this, 0x00, sizeof(*this));
+
+        mVersion = CeLoginInvalidVersion;
+        mType = AcfType_Invalid;
+    }
+
+    AcfVersion  mVersion;
+    AcfType     mType;
+    uint64_t    mExpirationTime;
+
+    union
+    {
+        struct
+        {
+            ServiceAuthority mAuth;
+        } mServiceFields;
+
+        struct
+        {
+            char mAdminAuthCode[AdminAuthCodeMaxLen];
+            uint64_t mAdminAuthCodeLength;
+        } mAdminResetFields;
+    };
+};
+
+/// @note This function will return failure if called with a V2 ACF
+[[deprecated]] CeLoginRc getServiceAuthorityV1(
```

Comment:
[[deprecated]] will throw a warning. OpenBMC will fail build on all warnings.

Reply:
They only throw the warning if they are called.

Reply:
The PAM specific service login function and the certificate manager code no longer use the V1 interfaces.

Reply:
I will be pushing up CLI changes in a separate review. I can mark the interfaces as deprecated at that point as well.